internal/configs/configurator.go

```diff
@@ -1428,6 +1439,8 @@ func (cnf *Configurator) AddOrUpdateSecret(secret *api_v1.Secret) string {
 		return cnf.addOrUpdateCASecret(secret)
 	case secrets.SecretTypeJWK:
 		return cnf.addOrUpdateJWKSecret(secret)
+	case secrets.SecretTypeOIDC:
+		return cnf.addOrUpdateOIDCSecret(secret)
```

Comment:
I think it's misleading to have a func called `addOrUpdateOIDCSecret` which doesn't add or update a secret.

Can we just do `return ""`/ `return "" # OIDC ClientSecret is not required on the filesystem, it is written directly to the config file.` here? And remove `addOrUpdateOIDCSecret`.

Reply:
Yeah I was on the fence about that too, I'll change it.